src/app/screens/Accounts/Detail/index.tsx

```diff
@@ -196,6 +176,14 @@ function AccountDetail() {
     }
   }
 
+  async function setLiquidEnable(value: boolean) {
+    await msg.request("liquid/setIsLiquidEnable", {
```

Comment:
We can use the existing accounts/edit action here. Note: I can change it when merging master into this branch because we also moved the bitcoin network setting to the account settings

Reply:
Maybe easier for merge to do not include accounts actions changes here ?

Reply:
I think it is ok, we will create a PR for you to review with the updates